core/rs/core/src/tableinfo.rs

```diff
@@ -27,6 +28,7 @@ use sqlite_nostd::ResultCode;
 use sqlite_nostd::Stmt;
 use sqlite_nostd::StrRef;
 
+const MAX_CL_CACHE_SIZE: usize = 1500;
```

Comment:
Please add an TODO that this should become configurable via the same mechanism as merge_equal_values.